assets/_sass/_structure.scss

```diff
@@ -433,6 +433,15 @@
     width: 100%;
   }
 
+  .global-footer {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: #ebebeb;
```

Comment:
@CharlRitterDev should this not be using colours defined in `_sass/default-variables.scss`?

Reply:
@kmorake 100% I just wanted a light grey to demo with, and to show options.
This will definitely need to be updated for styling before being released.

Reply:
Jip, although ideally it lives somewhere both themes & FE ingest it. One day